manifests/yum/repo/internet_simp_server.pp

```diff
@@ -26,7 +26,7 @@
     enabled         => 1,
     enablegroups    => 0,
     gpgcheck        => 1,
-    gpgkey          => 'https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP',
+    gpgkey          => 'https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP-6',
```

Comment:
This is not correct and needs to be changed to the appropriate array with both keys.